docs/cloud/satellites/byoc/byoc.md

```diff
@@ -0,0 +1,70 @@
+
+Here is how Self-hosted satellites compare to BYOC:
+
+| Similarities                                                     | Differences                                                                            |
+|------------------------------------------------------------------|----------------------------------------------------------------------------------------|
+| ✅ The satellite lives in your infrastructure, next to your tools | ❌ You cannot manually provision satellites                                             |
+| ✅ You pay for the usage the satellite incurs                     | ❌ You must allow Earthly access to at least some portion of an AWS account you control |
+|                                                                  | ❌ You pay the cloud provider, not Earthly, for the compute usage                       |
+
+## Installation
+
+### Step 1: Configure Your Cloud Provider
+
+Right now, BYOC Satellites are only supported in AWS. See our [AWS CloudFormation](./aws.md) instructions to provision the required AWS resources.
+
+
+### Step 2: Install In Earthly
+
+After installing the required resources within your cloud provider; you must complete the installation by telling Earthly about your new configuration. Run:
+
+```shell
+earthly cloud install <name>
+```
+
+Where `<name>` is the name of the Installation, as specified by your cloud provider. Assuming it reports the status as `Green`, you should be good to go!
+
+
+### Step 3: Networking
+
+To use a satellite created by BYOC, you'll need to configure your networking (usually a VPN) to ensure access. We have guides to enable BYOC on VPNs for the following VPN providers:
+* [Tailscale](./tailscale.md)
+
+### Step 4. Test Drive
+
+Now that you have your cloud installation configured in your cloud provider and Earthly, its time to take it for a test drive!
+
+First, make sure you can see the cloud you just installed by running `earthly cloud list`, which lists all the cloud installations within your organization. Your output should look something like this:
+
+```shell
+❯ earthly cloud list
+   NAME           SATELLITES  STATUS          
+   my-new-cloud   0           Green  
+*  earthly-cloud  2           Green  
+```
+The `*` indicates the default cloud that will be used when launching satellites within your organization, unless otherwise specified.
+
+
+{% hint style='info' %}
+Note that the `earthly-cloud` installation is a special cloud present in all organizations. Satellites within this cloud are managed within Earthly's cloud, by Earthly. You can change back to use Earthly-managed satellites at any time by running the `earthly cloud use earthly-cloud` installation.
+{% endhint %}
+
+To test-launch a new satellite within the cloud, run `earthly satellite launch --cloud <name> my-byoc-sat`. This will launch a new satellite using your cloud. Assuming that works, kick the tires by trying to run one of your builds on it!
```

Comment:
```suggestion
To test-launch a new satellite within your new cloud, run `earthly satellite launch --cloud <name> my-byoc-sat`. This will launch a new satellite using your cloud. Assuming that works, kick the tires by trying to run one of your builds on it!
```